ocaml/sdk-gen/common/CommonFunctions.mli

```diff
@@ -137,3 +137,12 @@ val session_id : param
 
 val objects : obj list
 (** Objects of api that generate SDKs. *)
+
+val get_published_release_for_param : string list -> string
```

Comment:
How about naming it as `published_release_of_param`?

Reply:
Nice, it's better.